### ISSUE TITLE
model: tensorflow: cnn: Transfer learning model

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#1280]
@shauryaDugar will be contributing a model based on mobilenet

**Comment by Onkar627** [comment]
Can I work on this? Also where should I add these models @pdxjohnny?

**Comment by programmer290399** [comment]
@Onkar627 Yes! Please see https://intel.github.io/dffml/master/contributing/dev_community.html#who-s-working-on-what. As a rule, we don't ask if we can do something, we state our intention to try to do it, then follow those guidelines.

**Comment by Onkar627** [comment]
Yeah, sure. Thanks!
